EightfoldAI/octuple
# fix: make stepper region focusable if scrollable

## SUMMARY:
In the case of onboarding flow, was getting axe devtools issue saying "Scrollable region must have keyboard access". So added a tabindex attribute to make the stepper region focusable to fix the issue.

## GITHUB ISSUE (Open Source Contributors)

## JIRA TASK (Eightfold Employees Only):
https://eightfoldai.atlassian.net/browse/ENG-85455

## CHANGE TYPE:


## TEST COVERAGE:


## TEST PLAN:
![image](https://github.com/user-attachments/assets/b87f92a9-ae6e-4552-b664-d9f83fe5cfbc)

- pull this locally and bring up storybook
- confirm that the stepper center region is focusable and user can scroll using the keyboard or not.
- confirm that other variants of the stepper component are still working as expected too.
